snuba/settings.py

```diff
@@ -5,23 +5,27 @@ def _load_settings(obj=locals()):
     main Snuba directory, or they can provide a full absolute path such as
     `/foo/bar/my_settings.py`."""
 
+    import importlib
+    import importlib.util
     import os
-    import imp
-
-    path = os.path.dirname(__file__)
 
     settings = os.environ.get('SNUBA_SETTINGS', 'base')
     if not settings.startswith('/') and not settings.startswith('settings_'):
-        settings = 'settings_%s' % settings
-    if not settings.endswith('.py'):
+        settings = '.settings_%s' % settings
+    elif not settings.endswith('.py'):
         settings += '.py'
 
-    settings = os.path.join(path, settings)
-    settings = imp.load_source('snuba.settings', settings)
+    if settings.startswith('/'):
```

Comment:
Rather than having two if statements, it'd probably be easier to read if you just consolidated. Especially since the `.py` extension can only be used when the path starts with `/` now.

Reply:
Makes sense, will see what I can do in accordance with my previous comment.